Update XSynth to 0.3.x

MyBlackMIDIScore opened the pull request.

KaleidonKep99 commented:
All good once again.

KaleidonKep99 commented:
Off topic, but did you deprecate xsynth-kdmapi in favor of the native XSynth implementation in OMv2?

MyBlackMIDIScore commented:
> Off topic, but did you deprecate xsynth-kdmapi in favor of the native XSynth implementation in OMv2?
> 
> EDIT: nvm you archived clib too 😕 EDIT-EDIT: nvm I see they're now part of the main xsynth codebase 👁️

Yeah at first I wanted to deprecate the kdmapi implementation because of OMv2, but Kaydax did not agree so we moved it back to the repo (plus I added config files and stuff).
And same with clib yeah, we moved it to the main xsynth repo and the binaries are released alongside kdmapi and xsynth-render.
